src/commands/toolbox/storage/withdraw.ts

```
@@ -0,0 +1,47 @@
+        const withdrawSpinner = ora(`Withdrawing ${flags.all ? 'all' : args.amount} from storage account...`).start()
+
+        if (flags.all) {
+            const balance = await storageProvider.getBalance()
+
+            await storageProvider.withdrawAll(balance)
+        } else {
+            await storageProvider.withdraw(sol(Number(args.amount)))
+        }
+
+        const newBalance = await storageProvider.getBalance()
+        withdrawSpinner.succeed('Funds withdrawn from storage account. New balance: ' + newBalance.basisPoints)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)